src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
```

Comment:
Shouldn't this just be %~dp0\..\..\..\..\ ? Again, better to not specify it if it's possible, we've made assumptions on the location of these scripts already